README.md

```diff
@@ -113,7 +113,9 @@ rgbd_obstacle_layer:
   observation_persistence: 0.0  #seconds
   max_obstacle_height:   2.0    #meters
   mark_threshold:        0      #voxel height
-  update_footprint_enabled: true
+  update_footprint_enabled: true 
+  footprint_projection_enabled: true #default off, recommended when using 3d IMU data. Uses tf2 to transform the footprint
```

Comment:
Set to `false` here, most people copy/paste this for use so it should be kept as a good default-centric baseline

Reply:
I think somewhere in the readme, we need to have some explanation of your relative filters + how to set it up, so that its clear for later users how to use it and why its useful